app/backend/src/couchers/servicers/gis.py

```diff
@@ -47,3 +47,25 @@ def GetCommunities(self, request, context):
                 content_type="application/json",
                 data=data,
             )
+
+    def GetPages(self, request, context):
+        with session_scope() as session:
+            # TODO: this might get very slow, very quickly
+            out = session.execute(
+                """
+            select json_build_object(
+                    'type', 'FeatureCollection',
+                    'features', json_agg(ST_AsGeoJSON(t.*, 'geom', 6)::json)
+                )
+            from (
+                select page_id as id, geom from page_versions where id in (select max(id) from page_versions group by page_id) and geom is not null
```

Comment:
Shouldn't the request argument be used the narrow down the results?

Reply:
This is just for the map. This gis stuff is a bit WIP at the moment, and will need to go through a few iterations as we get more data.